README.md

```diff
@@ -1,14 +1,246 @@
+    <p>At its core, SWA simplifies the process of connecting dApps with users' preferred wallets, offering seamless integration with over a dozen Solana wallets right out of the box. By leveraging SWA, developers can eliminate the complexities associated with individually supporting each wallet, thus enhancing accessibility and user experience. </p>
+  
+####Key features and benefits of SWA include: 
+- Open Source and Solana Labs Support: SWA is an open-source project actively supported by Solana Labs, ensuring continuous improvement and community-driven development.
+- Multi-Wallet Support: With SWA, developers can effortlessly accommodate users utilizing various Solana wallets, eliminating the need to manage multiple integrations separately.
+- Customizable UI: SWA provides an off-the-shelf customizable user interface, enabling developers to tailor the wallet integration experience to align with the aesthetic and functional requirements of their dApps.
+- Anchor Integration: SWA seamlessly integrates with Anchor, a comprehensive development framework for building Solana dApps, offering enhanced functionality and compatibility.
+- Key Functionality: SWA offers essential functionalities such as Connect, Disconnect, and Auto-connect, streamlining the interaction between dApps and users' wallets.
+- Support for Multiple Environment and Front-End Frameworks: SWA is designed to work seamlessly with various front-end frameworks and environment such as mobile and gaming development (unit), offering flexibility and compatibility across different development environments. 
+ </div>
+
+
+
+<!--PROJECT USE LINKS -->
+<p align="center"> 
+    <a href="#"><strong>View demo</strong> </a>
+     ·
+    <a href="#"><strong>Report bug</strong> </a>
+    ·
+    <a href="#/df"><strong>Request feature</strong> </a>
+     ·
+    <a href="#/df"><strong>FAQ</strong> </a>
+</p>
+
+
+<!--PROJECT TABLE OF CONTENTS -->
+
+###Table of Contents 
+ <ul>
+    <li>
+    Installation & Setup
+    <ul>
+    <li>
+    <a href="#ReactJS"> ReactJS </a>
+     </li>
+     <li>
+    <a href="https://github.com/lorisleiva/solana-wallets-vue"> VueJS </a>
+     </li>
+      <li>
+    <a href="https://github.com/svelte-on-solana/wallet-adapter"> Svelte </a>
+     </li>
+      <li>
+    <a href="https://github.com/magicblock-labs/Solana.Unity-SDK"> Unit-SDK </a>
+     </li>
+     </ul>
+     </li>
+      <li><a href="https://github.com/anza-xyz/wallet-adapter/blob/master/FAQ.md">FAQ</a></li>
+      <li><a href="https://github.com/anza-xyz/wallet-adapter/blob/master/PACKAGES.md">Packages</a></li>
+      <li><a href="https://github.com/anza-xyz/wallet-adapter/blob/master/BUILD.md">Build from source</a></li>
+      <li><a href="#Resource">Resource</a></li>
+    
+  </ul>
```

Comment:
I don't think we should add a table of contents. [Github does this for us if we use Markdown headings](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)